docs/user/dashboard/vega-reference.asciidoc

```diff
@@ -444,6 +444,8 @@ kibanaSetTimeFilter(start, end)
       hideWarnings: true
       // Vega renderer to use: `svg` or `canvas` (default)
       renderer: canvas
+      // If true, Restores Vega signal values on refresh
+      restoreSignalValuesOnRefresh: true
```

Comment:
Please indicate that `false` is the default value.

Also, can you add a sentence in the previous section about Kibana-specific functions like `kibanaAddFilter`? Something like this language:

> Kibana has extended the Vega expression language with these functions. These functions will trigger new data to be fetched, which by default will reset Vega signals. To keep signal values set `restoreSignalValuesOnRefresh: true` in the Vega config.